Tests/ArgumentParserUnitTests/HelpGenerationTests.swift

```diff
@@ -798,3 +798,54 @@ extension HelpGenerationTests {
       """)
   }
 }
+
+extension HelpGenerationTests {
+  private struct WideHelp: ParsableCommand {
+    @Argument(help: "54 characters of help, so as to wrap when columns < 80")
+    var argument: String?
+  }
+  
+  func testColumnsEnvironmentOverride() throws {
+    #if os(Windows) || os(WASI)
+    throw XCTSkip("Unsupported on this platform")
+    #endif
+    
+    unsetenv("COLUMNS")
```

Comment:
Do we need to `defer { unsetenv("COLUMNS") }` to avoid this test colliding with other tests which assert on help screens?

I'm actually wondering if this is safely testable when swift test --parallel is used...

Reply:
Oof, that's a good point - Definitionally, it can't really be parallel-safe, in that environment variables themselves are mutable shared global state. The best I think I can do is lessen the amount of variance in the `COLUMNS` value (such as only setting it to 80 and 81, keeping it as close as possible to the default).

Reply:
I'm wondering if the best solution is to just not set the env var and skip test coverage. I know thats a gross solution, but I'd much rather have 100% reliable tests than missing test coverage (imo)

Reply:
@natecook1000

Reply:
I have a partial workaround - I can use `CommandLine.arguments` to detect when parallel testing is in effect (the commandline signatures are unique on both macOS and Linux), and use that to skip the test. The caveat is that it only works for `swift test`; I'd have to skip it unconditionally in Xcode.

Reply:
Update: I tried this solution and it worked in multiple Swift versions on both platforms, so I pushed the code.

Reply:
Alternatively, could we have this test be the only place that checks that the default 80 column width is used, and just specify exact widths in all the other help generation tests?